src/test/java/com/microsoft/sqlserver/jdbc/datatypes/TVPWithSqlVariantTest.java

```diff
@@ -13,10 +13,7 @@
 import java.sql.SQLException;
 import java.sql.SQLTimeoutException;
 
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Tag;
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.*;
```

Comment:
Use specific classes instead of * import.

Reply:
Hmm strange, must've been the auto import